Cargo.toml

```diff
@@ -4,19 +4,24 @@ version = "0.1.0"
 authors = ["Parker J. Rule <parker.rule@tufts.edu>"]
 edition = "2018"
 
+[package.metadata.maturin]
+python-source = "python_src"
+
 [dependencies]
-serde = { version = "1.0.64", features = ["derive"] }
-serde_json = "1.0.64"
-clap = "2.33.3"
-rand = { version = "0.8.3", features = ["small_rng"] }
-mimalloc = { version = "0.1.25", default-features = false }
+clap = "2.33.4"
 crossbeam-channel = "0.5.0"
 crossbeam = "0.8.0"
-sha3 = "0.9.1"
-petgraph = "0.6.0"
+mimalloc = {version = "0.1.27", default-features=false}
 ndarray = { version = "0.14", optional = true }
 ndarray-linalg = { version = "0.13", features = ["openblas-system"], optional = true }
 pcompress = "1.0.6"
+petgraph = "0.6.0"
```

Comment:
I sorted these, and the IDE bumped some of them. I'm happy to revert if this fluff bugs you. The import new stuff is the addition of pyo3 and snafu.